application-analytics-api/src/main/java/com/xwiki/analytics/configuration/AnalyticsConfiguration.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.analytics.configuration;
+
+import org.xwiki.component.annotation.Role;
+import org.xwiki.stability.Unstable;
+
+/**
+ * Analytics Configuration.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Role
+@Unstable
+public interface AnalyticsConfiguration
+{
+    /**
+     * @return Returns the address where the requests will be made.
```

Comment:
You don't need to duplicate `return`. So just `@return the address where the requests will be made.` (below as well). Also, you should start with lower case, since you can see it as  `return [something]`